app/views/posts/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="post">
+    <h4> Post: <%= @post.title %> </h4>
+    <p class='count'>Comments: <%= @post.comments.count %> | Likes: <%= @post.likes.count %></p>
+    <p><%= @post.text %></p>
+  </div>
+  <div class="comments">
+    <% @post.comments.each do |comment| %>
+      <div class="comment">
+        <p><strong><%= comment.user.name %> :</strong> <%= comment.text %></p>
+      </div>
+    <% end %> 
+  </div>
+</div>
```

Comment:
- [ ] Kindly improve the width of the elements to match [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAACNDGUTA7FY2ZKR4VKWQBBT2ZNQ6DWQ).

  ![image](https://github.com/Obimbo07/tdn/assets/36542986/a6a2da99-0f41-4da1-a2a7-f99381ab5618)

Reply:
Fixed.